extensions/wikia/Recirculation/templates/client/premiumRail.mustache

```diff
@@ -17,6 +17,19 @@
 						{{#thumbnail}}
 							<img src="{{thumbnail}}" alt="{{title}}" itemprop="thumbnail">
 						{{/thumbnail}}
+						{{#isVideo}}
+							<div class="video-icon-container">
+								<svg id="wds-player-icon-play" class="wds-player-icon-play-tiny" viewBox="0 0 180 180" width="100%" height="100%">
```

Comment:
better ideas about how to include wds svg into mustache template rendered by js are very welcome